install-panel.sh

```diff
@@ -923,10 +1055,16 @@ main() {
       exit 1
     fi
   fi
+  
+  # choose version/branch
+  choose_branch
```

Comment:
I think it should be somehow refactored that it checks if the OS is compatible and exit's before letting the user choose branch. It is kinda strange if it first lets you choose a branch and then exit's because the OS is not compatible.

Reply:
Fixed.

Reply:
No it is not fixed. The script still exists in `configure_php` (which btw is a bad name for a function which exits) which is after choosing the branch.